introduction.md

```diff
@@ -2,3 +2,7 @@ good to eat
 i love you
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?